tests/test_orbit_phase.py

```diff
@@ -69,6 +71,16 @@ def test_J1855_zero_ecc(self):
         phs3 = self.mJ1855.orbital_phase(self.mJ1855.T0.value + 0.1, anom="true")
         assert phs3 == phs1, "True anom != Mean anom"
 
+    def test_J1855_ell1(self):
+        log = logging.getLogger("test_J1855_ell1")
+        phs1 = self.mJ1855ell1.orbital_phase(self.mJ1855ell1.TASC.value, anom="mean")
+        assert phs1.value == 0.0, "Mean anom != 0.0 at TASC as value"
+        phs1 = self.mJ1855ell1.orbital_phase(self.mJ1855ell1.TASC, anom="mean")
+        assert phs1.value == 0.0, "Mean anom != 0.0 at TASC as MJDParam"
+        phs1 = self.mJ1855.orbital_phase(self.mJ1855ell1.TASC.value + 0.1, anom="mean")
+        phs2 = self.mJ1855.orbital_phase(self.mJ1855ell1.TASC.value + 0.1, anom="ecc")
+        assert phs2 != phs1, "Eccen anom == Mean anom"
```

Comment:
How different should they be? Floating-point equality is not nice.

Reply:
Yeah, I don't actually know the right answer for that.  All I'm really testing there is that they are different.  Because there have been mistakes in the past where people have computed all of the anomalies for ELL1 as being the same.  So I'm not really sure what the right thing to do here (or where I would get the values to test against).

Reply:
I mean 1e-3*eccentricity should probably cover it - I'm just concerned that someone might compute the "same" anomaly using slightly different order of operations and get a few ULP difference.